Dockerfile

```diff
@@ -15,10 +14,15 @@
 
 COPY --from=builder /app .
 
-RUN pip install --upgrade pip && \
-    pip install -r requirements.txt
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    && pip install --upgrade pip \
+    && pip install mysqlclient \
+    && pip install -r requirements.txt
```

Comment:
Ensure that the `requirements.txt` file is present in the context of the Docker build. Otherwise, the `pip install -r requirements.txt` command will fail.